app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/AddPictureFragment.kt

```diff
@@ -43,10 +47,10 @@ class AddPictureFragment : Fragment() {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? {
+    ): View {
         parentActivity = requireActivity()
         _binding = FragmentAddPictureBinding.inflate(inflater, container, false)
-        return binding.root
+        return _binding!!.root
```

Comment:
Why is this necessary? Good that you changed it everywhere if so, then it's consistent.

Reply:
hum to be honest I am not 100% sure to fully understand but if you want to undertsand, I think you should look at the Code review about the AddPictureFragment.

Reply:
short answer: `binding` is an alias for `_binding!!` and it comes from Google sample code somewhere.

Longer answer: it's so that the non-null assertion can be implicit and assume that it's good as long as the fragment hasn't been destroyed.

Remark: this change is wrong and should be reversed.

Reply:
Okay ! I will change all my modifications and leave it as it was. Thanks you for the explanation !